storybook/src/admin/form/AllFieldComponents.tsx

```diff
@@ -41,7 +41,7 @@ function Story() {
                             <Card variant="outlined">
                                 <CardHeader title="Common Field-Components" titleTypographyProps={{ variant: "h3" }} />
                                 <CardContent>
-                                    <TextField name="text" label="Text" fullWidth />
+                                    <TextField name="text" label="Text" fullWidth type="email" />
```

Comment:
I assume this wasn't intentional 😄 

```suggestion
                                    <TextField name="text" label="Text" fullWidth />
```

Reply:
Oh no :( 
Sorry for that!